src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplateBase/help-securityOpts.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    A list of new line separated security options.
```

Comment:
Please expand this help.
Unless I've misunderstood, I think that the syntax expected here is that each line specified here is equivalent to passing one `--security-opt` argument to `docker run` - if that's true, please expand the help text to make this clear (and if not, please explain what it really is).
Maybe even include an example and/or a URL link to https://docs.docker.com/engine/reference/run/#security-configuration too.
If it's clearly documented then we won't end up getting mistaken "bug reports" from people who didn't understand it :grin:

i.e. please use this file to remove _any_ excuses for people to raise a bug report about this...